android-auto/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mapbox.onetap.androidauto">
```

Comment:
```suggestion
    package="com.mapbox.androidauto">
```

Reply:
```
package="com.mapbox.examples.androidauto">
```

we go through this, we probably want to do